app/controllers/api/v1/subjects_controller.rb

```diff
@@ -1,4 +1,14 @@
 class Api::V1::SubjectsController < ApplicationController
+  def index
+    subjects = Subject.order(created_at: :desc)
+
+    if subjects.any?
```

Comment:
Could we change `any` to `exist`

```suggestion
    if subjects.exists?
```